app/serializers/user.js

```diff
@@ -1,6 +1,11 @@
 import ApplicationSerializer from './application';
 
-const SKIP_SERIALIZED = ['loginActivity', 'account', 'person'];
+const SKIP_SERIALIZED = [
+  'loginActivity',
+  'account',
+  'secretaryForAgendas',
+  'secretaryForDecisions',
+];
```

Comment:
Are we sure `person` doesn't need to be skipped here? (Not sure what this does actually)

Reply:
As correctly pointed out by @sergiofenoll , user has no `secretaryForAgendas` nor `secretaryForDecision` relation